posthog/test/test_migration_0219.py

```diff
@@ -15,6 +14,8 @@ def setUpBeforeMigration(self, apps):
         Insight = apps.get_model("posthog", "Insight")
         Tag = apps.get_model("posthog", "Tag")
         TaggedItem = apps.get_model("posthog", "TaggedItem")
+        Team = apps.get_model("posthog", "Team")
```

Comment:
This migration test was using the normal `Team` import, but because it's a migration test, it should be calling `apps.get_model("posthog", "Team")` instead to get the Team object at the point in time of the migration.

The test was breaking after the changes in this PR because when the new Team is deleted, it triggers the cascade to delete `InsightViewed`, but `InsightViewed` doesn't exist at the time of this migration.